cmd/validate.go

```diff
@@ -74,4 +125,9 @@ func init() {
 		"state", "s", []string{"kong.yaml"}, "file(s) containing Kong's configuration.\n"+
 			"This flag can be specified multiple times for multiple files.\n"+
 			"Use '-' to read from stdin.")
+	validateCmd.Flags().BoolVar(&validateCmdKongAPI, "use-kong-api",
+		false, "whether to leverage Kong's API to validate configuration.")
+	validateCmd.Flags().StringSliceVarP(&validateCmdKongAPIEntity,
+		"entity", "e", []string{}, "entities to be validated via Kong API "+
+			"(e.g. -e entity1 -e entity2)")
```

Comment:
That's us asking the user to make one more decision. Are there any specific gains that you see here with this option?
A simpler product experience would be:
- deck validate -> validates using only the json schema in place (what decK does today in main branch)
- deck validate --use-kong-api (we will need to explore a better name for the flag) ->  this validates the entities within the file against Kong's Admin API. See the example in the comment I left as to what validation means in this context.

Thoughts?

Reply:
I totally see your point! I got misleaded when I read the docs in the first place as I hadn't realize the entity should have been passed in the body, which led me to this poor implementation. Thanks for the pointers, I'll rework this!